assets/ics/irc-meeting.ics

```diff
@@ -0,0 +1,180 @@
+BEGIN:VCALENDAR
+PRODID:-//Google Inc//Google Calendar 70.9054//EN
+VERSION:2.0
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+X-WR-CALNAME:Bitcoin-dev
+X-WR-TIMEZONE:Atlantic/Reykjavik
+X-WR-CALDESC:Bitcoin development meetings
+BEGIN:VTIMEZONE
+TZID:Atlantic/Reykjavik
+X-LIC-LOCATION:Atlantic/Reykjavik
+BEGIN:STANDARD
+TZOFFSETFROM:+0000
+TZOFFSETTO:+0000
+TZNAME:GMT
+DTSTART:19700101T000000
+END:STANDARD
+END:VTIMEZONE
+BEGIN:VEVENT
+DTSTART:20200811T150000Z
+DTEND:20200811T160000Z
+RRULE:FREQ=WEEKLY;WKST=MO;UNTIL=20210322T235959Z;INTERVAL=2;BYDAY=TU
+EXDATE:20201229T150000Z
+DTSTAMP:20250403T004023Z
+UID:6j5og22fjsf55qlecj5d5lslik@google.com
+CREATED:20200806T195438Z
+LAST-MODIFIED:20210303T195953Z
+SEQUENCE:2
+STATUS:CONFIRMED
+SUMMARY:Bitcoin Core P2P meeting

```

Comment:
Do these P2P meetings still happen?

Reply:
I don't think so, but I believe these lines in the ics file refer to one time historical events, not recurring events, they don't show up in modern times when I import the ical file, and I can go back to the listed dates (In August of 2020) and see them.

Reply:
Yes these refer to historical events